### PR TITLE
[VIS] Gamestate Roles

### PR DESCRIPTION
**Pull request by olly-larkin**
# Summary

Added judge, speaker, and president IDs to gamestate and updated orchestration to use these

## Additional Information

Initial visualisation design:

133659752_240720567415686_4583779582207912922_n

## Test Plan

image

screenshot of `output.json` showing it includes the relevant fields

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: internal/server/server.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ func createSOMASServer(
 			Environment:    disasters.InitEnvironment(clientIDs, gameConfig.DisasterConfig),
 			DeerPopulation: foraging.CreateDeerPopulationModel(gameConfig.ForagingConfig),
 			IIGOHistory:    []shared.Accountability{},
+			SpeakerID:      0,
+			JudgeID:        1,
+			PresidentID:    2,
```
Comment (lhl2617) [inline internal/server/server.go]:
```suggestion
			SpeakerID:      shared.Team1,
			JudgeID:        shared.Team2,
			PresidentID:    shared.Team3,
```

**Review by lhl2617 — APPROVED** [review]
image

**Review by Darrekt — APPROVED** [review]
What visualisation purpose do you have in mind wrt these changes?

**Comment by lhl2617** [comment]
can you also show a screenshot of `output.json` in your test plan showing that these fields are indeed output? thanks

**Comment by olly-larkin** [comment]
> What visualisation purpose do you have in mind wrt these changes?

Was planning on something like this but very open to suggestions...

133659752_240720567415686_4583779582207912922_n

**Comment by lhl2617** [comment]
> > What visualisation purpose do you have in mind wrt these changes?
> 
> Was planning on something like this but very open to suggestions...
> 
> 133659752_240720567415686_4583779582207912922_n

that's LIT

put that in the PR desc

**Pull request merged**